inst/private/python_header.py

```diff
@@ -241,7 +241,7 @@ def octoutput(x, et):
 
 
 try:
-    def make_matrix_or_array(ls_of_ls, dbg_no_array=True):
+    def make_matrix_or_array(ls_of_ls, dbg_no_array=False):
         # should be kept in sync with the same function
         # defined in inst/private/python_ipc_native.m
```

Comment:
I think we should also change `dbg_no_array` from `True` to `False` in `inst/private/python_ipc_native.m`.

LGTM otherwise as this test (for `@sym/and`)

```
%!test
%! % array
%! w = [t t f f];
%! z = [t f t f];
%! assert (isequal (w & z, [t f f f]))
```

now passes!

Reply:
Resolved in 37724d624c6952669709c85e058f3e0fc72b604c